.travis.yml

```diff
@@ -4,6 +4,7 @@ rvm:
   - 2.3.8
   - 2.4.5
   - 2.5.3
+  - 2.6.0
 before_install:
   - gem update --system
   - gem install bundler
```

Comment:
We could temporarily change this to install Bundler version 1, until the next `bundler-audit` version is released
https://github.com/rubysec/bundler-audit/pull/203

```
- gem install bundler --version 1.17.3
```

Reply:
Actually, given the new Rubygems includes Bundler, we don't need this line.